File: app/src/main/java/br/univali/contacts/ContactListFragment.java
package br.univali.contacts;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.LinearLayoutManager;
import br.univali.contacts.databinding.FragmentContactListBinding;

import java.util.ArrayList;
import java.util.List;

public class ContactListFragment extends Fragment {
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        FragmentContactListBinding binding = FragmentContactListBinding.inflate(inflater, container, false);
        binding.recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
        List<Contact> contacts = new ArrayList<>();
        binding.recyclerView.setAdapter(new ContactListAdapter(contacts));
        return binding.getRoot();
    }
}

File: app/src/main/java/br/univali/contacts/ContactListFragment.kt
package br.univali.contacts

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.annotation.NonNull
import androidx.fragment.app.Fragment
import androidx.fragment.app.viewModels
import androidx.recyclerview.widget.LinearLayoutManager
import br.univali.contacts.databinding.FragmentContactListBinding

class ContactListFragment : Fragment() {
    private val viewModel: ContactListViewModel by viewModels()
    private lateinit var binding: FragmentContactListBinding

    override fun onCreateView(
        @NonNull inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        binding = FragmentContactListBinding.inflate(inflater, container, false)
        binding.recyclerView.layoutManager = LinearLayoutManager(context)
        viewModel.contacts.observe(viewLifecycleOwner) {
            binding.recyclerView.adapter = ContactListAdapter(it)
        }
        return binding.root
    }

    override fun onResume() {
        super.onResume()
        viewModel.findAll(requireContext())
    }
}
